.github/workflows/ci_tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
3.7 + 3.9 should suffice, no need to test all intermediate versions usually

Reply:
Since tests here are pretty fast, I'd prefer going for all three.

Reply:
:+1: